superset/viz.py

```diff
@@ -306,7 +306,8 @@ def data(self):
     def get_csv(self):
         df = self.get_df()
         include_index = not isinstance(df.index, pd.RangeIndex)
-        return df.to_csv(index=include_index, encoding="utf-8")
+        return df.to_csv(index=include_index, encoding=str(config.get('CSV_EXPORT')['ENCODING']),
```

Comment:
Why casting to `str`?

Reply:
To re-cast the string, otherwise I get TypeError: "delimiter" must be string, not unicode with python 2.x
If you have a better solution, I can change that (or maybe add comment to clarify inside the code ?)